test/packages/bad_var_migrate_from/docs/README.md

```
@@ -0,0 +1,101 @@
+
+## Overview
+{{/* Complete this section with a short summary of what data this integration collects and what use cases it enables */}}
+The New Package integration for Elastic enables collection of ...
+This integration facilitates ...
+
+### Compatibility
+{{/* Complete this section with information on what 3rd party software or hardware versions this integration is compatible with */}}
+This integration is compatible with ...
+
+### How it works
+{{/* Add a high level overview on how this integration works. For example, does it collect data from API calls or recieving data from a network or file.*/}}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)